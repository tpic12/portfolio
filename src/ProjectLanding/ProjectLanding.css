.project-landing-section {
  width: 100vw;
  height: 100vh;
  background-color: #ededed;
  display: flex;
  flex-direction: column;
  /* margin-top: 100px; */
  justify-content: center;
  align-items: center;
  /* padding-top: 50px; */
}

.project-landing-title {
  width: 100vw;
  height: 75vh;
  color: rgb(62, 72, 117);
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: rgb(62, 72, 117);
  border-radius: 0 400px 400px 0;
  border-top: 5px solid white;
  border-right: 5px solid white;
  border-bottom: 5px solid white;
}

.project-landing-title svg {
  color: white;
  position: relative;
  left: 55px;
  animation: bounceX 3s ease-out;
  animation-iteration-count: infinite;
}

.project-landing-circle {
  border-radius: 50%;
  width: 200px;
  height: 200px;
  background-color: white;
  display: flex;
  justify-content: center;
  align-items: center;
}

@keyframes bounceX {
  0% {
    transform: translateX(0)
  }

  50% {
    transform: translateX(-80%)
  }

  100% {
    transform: translateX(0);
  }
}

@media screen and (min-width: 720px) {
  .project-landing-title {
    border-radius: 0;
    border: none;
    background-color: #ededed;
  }

  .project-landing-title svg {
    left: -80px;
    color: white;
  }

  .project-landing-circle {
    width: 700px;
    height: 400px;
    border-radius: 200px;
    background-color: rgb(62, 72, 117);
  }

  .project-landing-circle h2 {
    font-size: 64px;
    color: white;
  }
}