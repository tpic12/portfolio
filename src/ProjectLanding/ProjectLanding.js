import React, { Component } from "react";
import "./ProjectLanding.css";

class ProjectLanding extends Component {
  render() {
    return (
      <section className="project-landing-section snap">
        <div className="project-landing-title">
          <div className="project-landing-circle">
            <h2>Projects</h2>
          </div>
        </div>
      </section>
    );
  }
}

export default ProjectLanding;
