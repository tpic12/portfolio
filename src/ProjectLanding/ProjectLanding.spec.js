import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ProjectLanding from "./ProjectLanding";

describe("<Landing />", () => {
  it("renders by default", () => {
    const wrapper = shallow(<ProjectLanding />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
