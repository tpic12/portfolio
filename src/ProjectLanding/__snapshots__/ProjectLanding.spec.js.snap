// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders by default 1`] = `
<section
  className="project-landing-section snap"
  id="project-landing-section"
>
  <div
    className="project-landing-title"
  >
    <div />
    <div
      className="project-landing-circle"
    >
      <h2>
        Projects
      </h2>
    </div>
    <a
      href="#qtrail-section"
    >
      <FontAwesomeIcon
        border={false}
        className=""
        fixedWidth={false}
        flip={null}
        icon="chevron-right"
        inverse={false}
        listItem={false}
        mask={null}
        pull={null}
        pulse={false}
        rotation={null}
        size="lg"
        spin={false}
        swapOpacity={false}
        symbol={false}
        title=""
        transform={null}
      />
    </a>
  </div>
</section>
`;
